### DISCUSSION TITLE
Privacy Badger : Test sites?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [EFForg/privacybadger#2814] [Q&A]
Hi.
Are there pages to test PB?
thx

**Comment by ghostwords** [comment]
Hi, I'm not sure what you are looking for exactly. Are you asking about [Privacy Badger's automated tests](https://github.com/EFForg/privacybadger/blob/master/doc/tests.md) or something else? Could you clarify?

**Reply by Clemens-Ratte-Polle** [reply]
Hi.
I meant to show a page with all the typical threats out there in the
internet that PB kills as a test drive.
All privacy threats are listet and shown if disabled or not.
So one can test his Browser with and without PB.
And you could show what PB is good for.

Also this would be fine to test all other privacy addons to see the
differences :)
There are pages like this:
[https://amiunique.org/fp](https://www.google.com/url?q=https%3A%2F%2Famiunique.org%2Ffp&sa=D&sntz=1&usg=AFQjCNFCDrRxHAKvKn_zAC-lOcx-RWxsZQ)
[https://privacy.net/analyzer/#pre-load](https://www.google.com/url?q=https%3A%2F%2Fprivacy.net%2Fanalyzer%2F%23pre-load&sa=D&sntz=1&usg=AFQjCNFwpbAWA40JAHLU9G-74w8HaflzDA)

Only an idea :)

**Reply by ghostwords** [reply]
Here is the [latest description of Privacy Badger's functionality](https://github.com/EFForg/privacybadger#privacy-badger):

>Privacy Badger is a browser extension that automatically learns to block invisible trackers. Instead of keeping lists of what to block, Privacy Badger automatically discovers trackers based on their behavior.
>
>Privacy Badger sends the [Global Privacy Control](https://globalprivacycontrol.org/) signal to opt you out of data sharing and selling, and the [Do Not Track](https://www.eff.org/issues/do-not-track) signal to tell companies not to track you. If trackers ignore your wishes, Privacy Badger will learn to block them.
>
>Besides automatic tracker blocking, Privacy Badger comes with privacy features like [click-to-activate replacements](https://privacybadger.org/#How-does-Privacy-Badger-handle-social-media-widgets) for potentially useful trackers (video players, comments widgets, etc.), and link cleaning on [Facebook](https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking) and [Google](https://www.eff.org/deeplinks/2018/10/privacy-badger-now-fights-more-sneaky-google-tracking).
>
>To learn more, see [the FAQ on Privacy Badger's homepage](https://privacybadger.org/#faq).

So basically, Privacy Badger is a content blocker with some additional features. Privacy Badger is primarily concerned with reducing [non-consensual third-party tracking](https://privacybadger.org/#How-does-Privacy-Badger-work). The "threats" are the commercial trackers.

To see the effect Privacy Badger has on your browsing, try visiting a few news sites (for example) with and without Privacy Badger. When you visit the news sites with Privacy Badger, click on Privacy Badger's icon in your browser toolbar and note how many different domains Privacy Badger reports as blocked.

Does this help?
